pegasus/sites.v3/code.org/public/about/advisors.md.erb

```diff
@@ -1,9 +0,0 @@
----
-title: Education Advisors
-nav: about_nav
-theme: responsive
----
-# Education Advisory Council
-Our Education Advisory Council is a group of educators with decades of experience in teaching Computer Science, across all grade levels - elementary, secondary, or at the university level.
-
-<%= view :about_advisors, people:DB[:cdo_leaders].where(kind_s:'education') %>
```

Comment:
Can we remove `about_advisors` too?

Reply:
oooo good catch! I'll remove this once we can pull staging again:)

Reply:
Removed!